src/prelude/prim.mo

```diff
@@ -335,7 +335,7 @@ func cyclesRefunded() : Nat {
 };
 
 func cyclesAccept<system>(amount : Nat) : Nat {
-  (prim "cyclesAccept" : <system>Nat -> Nat)(amount);
+  (prim "cyclesAccept" : Nat -> Nat)(amount);
```

Comment:
Some cleanup :)

Reply:
Yep. Oh, maybe there is another one or two. Lemme check...

Reply:
Done